src/main/java/FileUtils/DecathlonDataReader.java

```diff
@@ -0,0 +1,9 @@
+package FileUtils;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.